man/systemd.network.xml

```diff
@@ -2296,6 +2296,16 @@ IPv6Token=prefixstable:2002:da8:1::</programlisting></para>
 
     <variablelist class='network-directives'>
 
+      <varlistentry>
+        <term><varname>RelayTarget=</varname></term>
+        <listitem>
+          <para>Takes an IPv4 address, which must be in the format
+          described in
+          <citerefentry project='man-pages'><refentrytitle>inet_pton</refentrytitle><manvolnum>3</manvolnum></citerefentry>.
+          Turns DHCP-server into a DHCP relay agent. See <ulink url="https://tools.ietf.org/html/rfc1542">RFC 1542</ulink>.
```

Comment:
Maybe "Turns this DHCP server into a DHCP relay agent." ?

Reply:
Addressed. Also I added a sentence which suggests usage of BindToInterface= option together with this one.